MapboxStatic/ClassicSnapshotOptions.swift

```diff
@@ -0,0 +1,213 @@
+#if os(OSX)
+    import Cocoa
+#elseif os(watchOS)
+    import WatchKit
+#else
+    import UIKit
+#endif
+
+/**
+ A structure that determines what a snapshot depicts and how it is formatted. A classic snapshot is made by compositing one or more [tile sets](https://www.mapbox.com/help/define-tileset/) with optional overlays using the [classic Mapbox Static API](https://www.mapbox.com/api-documentation/?language=Swift#static-classic).
```

Comment:
We’d have to tread carefully, because the component that translates Swift into Objective-C headers (and thus reStructuredText into Doxygen) is a bit fragile. Objective-C bridging has broken in the past due to something as innocuous as a blank line between `- parameter` and `- note`.